jdi-light-angular-tests/src/test/java/io/github/epam/angular/tests/elements/complex/RadioButtonTests.java

```diff
@@ -20,15 +20,109 @@ public void before() {
     }
 
     @Test
-    public void basicTest() {
-        basicRadioButtons.is().displayed();
-        firstBasicRadioButton.is().displayed();
-        secondBasicRadioButton.is().displayed();
+    public void displayedBasicRadioButtonTest() {
+        radioButtonSection.basicRadioButtons.is().displayed();
+        radioButtonSection.firstBasicRadioButton.is().displayed();
+        radioButtonSection.secondBasicRadioButton.is().displayed();
     }
 
     @Test
-    public void getTextTest() {
-        assertEquals(firstBasicRadioButton.getText(), "Option 1");
-        assertEquals(secondBasicRadioButton.getText(), "Option 2");
+    public void displayedSeasonRadioButtonTest() {
+        radioButtonSection.seasonRadioButtons.is().displayed();
+        radioButtonSection.winterNGModelRadioButton.is().displayed();
+        radioButtonSection.springNGModelRadioButton.is().displayed();
+        radioButtonSection.summerNGModelRadioButton.is().displayed();
+        radioButtonSection.autumnNGModelRadioButton.is().displayed();
     }
+
+    @Test
+    public void nameBasicRadioButtonTest() {
+        radioButtonSection.firstBasicRadioButton.has().text("Option 1");
+        radioButtonSection.secondBasicRadioButton.has().text("Option 2");
+    }
+
+    @Test
+    public void onlyOneOptionOneButtonEnabledTest() {
+        String enableBasicButton = "mat-radio-button mat-accent cdk-focused cdk-mouse-focused mat-radio-checked";
+        String disableBasicButton = "mat-radio-button mat-accent";
```

Comment:
Could you please move this two variables to class variables, you duplicate them in two methods.